src/dotnet/AutoRest.Core/Model/CodeModel.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
file not required

will stop here for now, please make sure you remove all files not required by the validator